static/linux/share/applications/webtorrent-desktop.desktop

```diff
@@ -0,0 +1,33 @@
+[Desktop Entry]
```

Comment:
Yes, they are almost the same. My first attempt was create a command line handler like the squirrel-win32 one, which can invoke the (un)install handler, but i need to pass a parameter to indicate that is called from the system installation (or detect the user invoking was the superadmin) instead of the standard user startup registration. Then, based on that change, modifiy the template variables and after that, modify the entire handlers.js script to return a callback to shutdown the app. Basically, It requires some work and testing in all the supported platforms and had no time, so, it was simplier to create the final 'packaged' version of the resources.